css/syntaxes.json

```diff
@@ -233,6 +260,9 @@
   "frequency-percentage": {
     "syntax": "<frequency> | <percentage>"
   },
+  "function-token": {
```

Comment:
`function-token` is a token type and out of scope of this dict.
Btw, it has another syntax

Reply:
So I should remove these then...? How should this be represented then in our data? Some kind of link to the spec definition of token types?

The syntax definition I included was basically a guess based on my knowledge of related syntax, but I didn't really know what I was doing.

Same comment for the other token types.

Reply:
Those tokens are defined in [CSS Syntax Module Level 3](https://drafts.csswg.org/css-syntax/) and shouldn't be presented in `syntax.json`. Because they are something like a generic, as `<string>` or `<number>` but a bit low-level.